src/theme/sass-partials/_buttons.scss

```diff
@@ -0,0 +1,21 @@
+
+.mes-primary-button {
```

Comment:
remove comments please :)

Reply:
Removed it but it's not relevant after all - so I'm just going to delete this file